gulpfile.js/premerge.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const simpleGit = require('simple-git/promise');
+
+
+function gitChanges() {
+	const git = simpleGit(__dirname);
+
+	console.log(__dirname);
+
+	return git.status().then(res => {
+		if (res.files.length > 0) {
+			console.log(res);
+			throw new Error('There are changes in the file system. Did you forget to run gulp?');
+		}
+	});
+}
+
+exports.premerge = gitChanges;
```

Comment:
```suggestion
module.exports = gitChanges;
```

Reply:
I didn't want to do this because it cuts off the path for additional exports later on.

It doesn't really matter right now but all other modules also export an object, so I wanted to keep it consistent.